module/state_synchronization/requester/execution_data_requester.go

```diff
@@ -395,7 +397,7 @@ func (e *executionDataRequester) processFetchRequest(ctx irrecoverable.SignalerC
 		ctx.Throw(err)
 	}
 
-	logger.Debug().Msg("Fetched execution data")
+	logger.Info().Msg("execution data fetched")
```

Comment:
nit: do we really want info? the log message doesn't prints out any details.
Dont know how frequently will we see this, but if it is very frequent then may be put it back to Debug?

Reply:
it will happen for every block, but it does print the blockID and other info